src/kata5.reachDestination.js

```diff
@@ -1,3 +1,13 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  let time = 0;
+  let timeRounded = 0.0;
```

Comment:
You need not create variables only to write over them straight away, could just be done at the same time. Also, all of these could be declared as `const` variables since they are not being reassigned
```
  const time = distance / speed;
  const timeRounded = Math.round(time * 2) / 2;
```